### DISCUSSION TITLE
Welcome to ytmusicapi Discussions!

### DISCUSSION THREAD

**Discussion by sigma67** [sigma67/ytmusicapi#361] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elsek95** [comment]
Hello. Can you share your a email or a social media link?

**Reply by sigma67** [reply]
It's in pyproject: https://github.com/sigma67/ytmusicapi/blob/master/pyproject.toml
